README.md

```diff
@@ -262,25 +262,57 @@ const MyApp = new Liftoff({
 });
 ```
 
-## launch(opts, callback(env))
-Launches your application with provided options, builds an environment, and invokes your callback, passing the calculated environment as the first argument.
+### prepare(opts, callback(env))
+
+Prepares the environment for your application with provided options, and invokes your callback with the calculated environment as the first argument.  The environment can be modified before using it as the first argument to `execute`.
+
+**Example Configuration w/ Options Parsing:**
 
-##### Example Configuration w/ Options Parsing:
 ```js
 const Liftoff = require('liftoff');
 const MyApp = new Liftoff({name:'myapp'});
 const argv = require('minimist')(process.argv.slice(2));
-const invoke = function (env) {
+const onExecute = function (env, argv) {
+  // Do post-execute things
+};
+const onPrepare = function (env) {
   console.log('my environment is:', env);
-  console.log('my cli options are:', argv);
   console.log('my liftoff config is:', this);
+  MyApp.execute(env, onExecute);
 };
-MyApp.launch({
+MyApp.prepare({
   cwd: argv.cwd,
   configPath: argv.myappfile,
   require: argv.require,
   completion: argv.completion
-}, invoke);
+}, onPrepare);
+```
+
+**Example w/ modified environment**
+
+```js
+const Liftoff = require('liftoff');
+const MyApp = new Liftoff({name:'myapp'});
```

Comment:
In this sample, the constructor options needs `configFiles` property.

```js
const Hacker = new Liftoff({  // MyApp -> Hacker
  name: 'hacker',   // myapp -> hacker
  configFiles: {
    '.hacker': {
      home: {  home: { path: '.', cwd: '~' } }
    }
  }
});
```
And config file names below uses `.hacker` or `hackerfile`, so `MyApp` should be changed to `Hacker`. (These are my mistakes. Sorry.)